data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/migrated/paths/_eth_v1_checkpoint_finalized_state.json

```diff
@@ -0,0 +1,56 @@
+{
+  "get" : {
+    "tags" : [ "Experimental" ],
+    "operationId" : "getFinalizedCheckpointState",
+    "summary" : "Get full BeaconState object for finalized checkpoint state",
+    "description" : "Returns full BeaconState object for a finalized checkpoint state from the Weak Subjectivity period. \nDepending on `Accept` header it can be returned either as json or as bytes serialized by SSZ.",
```

Comment:
We'd want to update this description to remove the reference to returning JSON as well.  And actually remove the application/json 200 OK response below too.